lib/rmt/downloader.rb

```diff
@@ -21,6 +21,13 @@ def initialize(repository_url:, destination_dir:, auth_token: nil, logger: nil,
     @concurrency = 4
     @auth_token = auth_token
     @logger = logger || Logger.new('/dev/null')
+    if ENV['RAILS_LOG_TO_JOURNALD'].present?
```

Comment:
maybe I could put the code somewhere else to make it DRY-ish ?

Reply:
Add an accessor in RMT::Config? Doesn't need to be written to the config file though.

Reply:
it should be more clean to make a logger class by advice from @ikapelyukhin